### PR TITLE
ADD Feature: Aggregate multi clusters resource status

### PR DESCRIPTION
**Pull request by aven-ai**
#### What type of PR is this?
kind/feature

#### What this PR does / why we need it:
ADD Feature: Aggregate child clusters resource status to parent

#### Special notes for your reviewer:
Add point
1. clusternet-agent: synchronize the k8s object of the child cluster to description.
2. clusternet-hub: Synchronize Description Status to Base, and aggregate Status to Subscription.
3. clusternet-scheduler: written scheduling to SubscriptionStatus condition.
4. clusternet-hub: Synchronize Subscription status to parent k8s object.

### CODE REVIEW

**Comment by aven-ai** [comment]
@dixudx Please take a look  about this feature, thanks.

File: pkg/scheduler/scheduler.go [hunk 1]
```diff
@@ -342,6 +348,9 @@ func (sched *Scheduler) scheduleOne(ctx context.Context) {
 
 			// Run "postbind" plugins.
 			sched.framework.RunPostBindPlugins(bindingCycleCtx, sub, targetClusters)
+
+			// TODO: update subscription condition
+			sched.updateSubscriptionSchedulingSuccess(sub)
```
Comment (dixudx) [inline pkg/scheduler/scheduler.go:353]: Scheduler should only focuses on scheduling. Better not add this here.

**Review by dixudx — CHANGES_REQUESTED** [review]
Better having a more concise implementation.

File: pkg/scheduler/scheduler.go (hunk 1 above)
Comment (aven-ai) [reply pkg/scheduler/scheduler.go:354]: > Scheduler should only focuses on scheduling. Better not add this here.

It's updating the scheduling result (success or failure) to SubscriptionStatus condition, I have made a new commit about this. 

[[Scheduling: Update the scheduling result (success or failure) to SubscriptionStatus condition](url)](https://github.com/clusternet/clusternet/pull/350/commits/a27c1321ee23d4143402e65a5f0bc2ee5289a201)

**Comment by aven-ai** [comment]
> I'd prefer a more concise implementation.

It is a good idea to make status sync as a controller in clusternet-hub. A little busy recently, consider doing it later.

**Comment by aven-ai** [comment]
Close this pr, move to #353

**Pull request closed without merging**